.github/workflows/test.yml

```diff
@@ -54,3 +54,8 @@ jobs:
       with:
         ruby-version: 3.2.2
         bundler-cache: true
+
+    - name: Run the bundle audit test
+      run: |
+        gem install bundler-audit
+        bundle audit check --update --ignore CVE-2024-25126 CVE-2024-26141 CVE-2024-26146
```

Comment:
@paul-mesnilgrente any reason for ignoring these three?

Reply:
Nope, it was from a copy paste, all removed :)